### PR TITLE
[dattri.matrics] add loo_corr

### PR DESCRIPTION
**Pull request by TheaperDeng**
## Description

Appears that `loo_corr` is missed. There is no api change since the docstring are quite clear.

### CODE REVIEW

File: test/dattri/metrics/test_loo.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
+"""Unit test for lds."""
+
+import unittest
+
+import torch
+
+from dattri.metrics.metrics import loo_corr
+
+
+class TestLDSFunction(unittest.TestCase):
```
Comment (jiaqima) [inline test/dattri/metrics/test_loo.py:10]: TestLDS -> TestLOO

File: test/dattri/metrics/test_loo.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    """Test LDS related functions."""
```
Comment (jiaqima) [inline test/dattri/metrics/test_loo.py:11]: LDS -> LOO

File: test/dattri/metrics/test_loo.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+    def test_loo(self):
+        """Test the lds metric."""
```
Comment (jiaqima) [inline test/dattri/metrics/test_loo.py:14]: lds -> loo_corr

**Review by jiaqima — APPROVED** [review]
Otherwise LGTM.

**Review by xingjian-zhang — APPROVED** [review]
LGTM besides the typo

**Pull request merged**